spring-pulsar-test/spring-pulsar-test.gradle

```diff
@@ -8,4 +8,8 @@ dependencies {
     implementation 'org.junit.jupiter:junit-jupiter-api'
     implementation 'org.testcontainers:pulsar'
     implementation 'org.testcontainers:junit-jupiter'
+    implementation 'org.awaitility:awaitility'
```

Comment:
I think we can move this back down to `testImplementation` now that we are not relying on it in our impl.